src/ImageSequencer.js

```diff
@@ -116,12 +116,18 @@ ImageSequencer = function ImageSequencer(options) {
   }
   
   function run(spinnerObj,t_image,t_from) {
-    let progressObj;
+    let progressObj,index=0;
+    
     if(arguments[0] != 'test'){
       progressObj = spinnerObj
-      delete arguments['0']    
+      delete arguments['0']
+      if(typeof progressObj == 'number') {
```

Comment:
Hmm, this is a bit hard to read - are you detecting if it's an index integer? If so can you explain a bit more in comments so it's more readable? Thanks!